editor/animation_track_editor.cpp

```diff
@@ -1799,7 +1800,7 @@ void AnimationTimelineEdit::update_values() {
 	}
 
 	editing = true;
-	if (use_fps && animation->get_step() > 0) {
+	if (use_fps && animation->get_step() > 0.0) {
 		length->set_value(animation->get_length() / animation->get_step());
 		length->set_step(FPS_DECIMAL);
```

Comment:
Need to double check that context wasn't lost when rebasing, as this looks potentially wrong. Gonna test real quick